src/db.py

```diff
@@ -27,40 +35,104 @@ def get_campus_eateries(data_json, refresh=False):
+                        Session.add_all(menu_items)
+                        Session.commit()
+
+        print("[{}] Fetching static campus eateries".format(datetime.now()))
+        static_json = requests.get(STATIC_EATERIES_URL).json()
+        static_eateries = campus_eateries
+
+        if refresh_campus:
+            print("[{}] Updating static campus eateries".format(datetime.now()))
+            static_eateries = parse_static_eateries(static_json)
+            Session.add_all(static_eateries)
             Session.commit()
 
-            for eatery_hour, menu_json in hours_and_menus:
-                categories_and_items = parse_menu_categories(menu_json, eatery_hour)
-                eatery_categories = (x[0] for x in categories_and_items)
-                Session.add_all(eatery_categories)
+        print("[{}] Updating static eatery hours and menus".format(datetime.now()))
+        for eatery in static_eateries:
+            if eatery.slug in STATIC_EATERY_SLUGS:
+                hours_and_menus = parse_static_op_hours(static_json, eatery)
+                eatery_hours = (x[0] for x in hours_and_menus)
+                Session.add_all(eatery_hours)
                 Session.commit()
 
-                for menu_category, items_json in categories_and_items:
-                    menu_items = parse_menu_items(items_json, menu_category)
-                    Session.add_all(menu_items)
+                for eatery_hour, menu_json in hours_and_menus:
+                    categories_and_items = parse_menu_categories(menu_json, eatery_hour)
+                    eatery_categories = (x[0] for x in categories_and_items)
+                    Session.add_all(eatery_categories)
                     Session.commit()
 
-        print("[{}] Updating collegetown".format(datetime.now()))
+                    for menu_category, items_json in categories_and_items:
+                        menu_items = parse_menu_items(items_json, menu_category)
+                        Session.add_all(menu_items)
+                        Session.commit()
+
+        Base.metadata.drop_all(bind=Engine, tables=[CollegetownEatery.__table__, CollegetownEateryHour.__table__])
+        Base.metadata.create_all(bind=Engine, tables=[CollegetownEatery.__table__, CollegetownEateryHour.__table__])
+        print("[{}] Fetching Collegetown eateries".format(datetime.now()))
         yelp_query = collegetown_search()
         collegetown_eateries = parse_collegetown_eateries(yelp_query)
         Session.add_all(collegetown_eateries)
         Session.commit()
 
+        print("[{}] Updating Collegetown eateries and hours".format(datetime.now()))
+        for eatery in collegetown_eateries:
+            hours = parse_collegetown_hours(yelp_query, eatery)
+            Session.add_all(hours)
+            Session.commit()
+
     except Exception as e:
         print("Data update failed:", e)
 
 
-start_update()
+start_update(True, True)
```

Comment:
does it need a function call? gunicorn file should take care of the initial call

Reply:
No, I just have it temporarily to run the function

Reply:
When we start the server do we want to refresh the eateries and recalculate the swipe data? If so, we should prob change the call to `start_update` in `gunicorn.py` to have the `True` args

Reply:
Also, if it's temporary, you should probably delete this line when you merge the changes

Reply:
Rip, I should've said that I already removed it